tests/foreman/api/test_subnet.py

```diff
@@ -278,6 +277,58 @@ def test_positive_subnet_parameters_override_impact_on_subnet(self):
         CaseLevel: System
         """
 
+        # Create subnet with valid parameters
+        parameters = [
+            {'name': gen_string('alpha'), 'value': gen_string('alpha')}
+        ]
+        org = entities.Organization().create()
+        loc = entities.Location(organization=[org]).create()
+        org_subnet = entities.Subnet(
+            location=[loc],
+            organization=[org],
+            subnet_parameters_attributes=parameters,
+        ).create()
+        self.assertEqual(
+            org_subnet.subnet_parameters_attributes[0]['name'],
+            parameters[0]['name']
+        )
+        self.assertEqual(
+            org_subnet.subnet_parameters_attributes[0]['value'],
+            parameters[0]['value']
+        )
+
+        # Create host with above subnet
+        host = entities.Host(
+            location=loc,
+            organization=org,
+            subnet=org_subnet,
+        ).create()
+        self.assertEqual(host.subnet.read().name, org_subnet.name)
+
+        # Change host parameter value and check that parameter override
+        # is successful without affecting subnet parameter
+        name = org_subnet.subnet_parameters_attributes[0]['name']
+        parameters = [{
```

Comment:
```suggestion
        new_parameter = [{
```
rename above i.e. `new_parameter` to make the difference with `parameters`.

Reply:
Done.